tests/integration/components/o-s-s/power-select-test.ts

```diff
@@ -0,0 +1,248 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, setupOnerror, findAll, click, typeIn, scrollTo } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+module('Integration | Component | o-s-s/power-select', function (hooks) {
+  setupRenderingTest(hooks);
+  const onSearch = sinon.stub();
+  const onChange = sinon.stub();
+
+  hooks.beforeEach(function () {
+    this.selectedItems = [];
+    this.items = [];
+    this.onSearch = onSearch;
+  });
+
+  module('it renders', () => {
+    test('without selected-item named block', async function (assert) {
+      setupOnerror((err: any) => {
+        assert.equal(err.message, '[component][OSS::PowerSelect] You must pass selected-item named block');
+      });
+
+      await render(hbs`
+        <OSS::PowerSelect @onSearch={{this.onSearch}}/>
+      `);
+    });
+
+    test('without option-item named block', async function (assert) {
+      setupOnerror((err: any) => {
+        assert.equal(err.message, '[component][OSS::PowerSelect] You must pass option-item named block');
+      });
+
+      await render(hbs`
+        <OSS::PowerSelect @onSearch={{this.onSearch}}>
+          <:selected-item as |selectedItem|>
+            {{selectedItem.name}}
+          </:selected-item>
+        </OSS::PowerSelect>
+      `);
+    });
```

Comment:
Move the first two tests to a 'It doesn't render' or 'It throws an error' module